fedimint-server/src/lib.rs

```diff
@@ -496,7 +496,7 @@ impl FedimintServer {
         }
 
         tokio::select! {
-            _peek = Pin::new(&mut self.tx_receiver).peek() => Ok(EpochTriggerEvent::NewTransaction),
+            _peek = Pin::new(&mut self.api_receiver).peek() => Ok(EpochTriggerEvent::NewTransaction),
```

Comment:
no longer correct?

Reply:
How so?  This is just a rename since we can receive any `ApiEvent`

Reply:
is `EpochTriggerEvent::NewTransaction` correct here?

Reply:
Ah I see, yes that should be renamed too.